src/checkers/htmlchecker.py

```diff
@@ -109,6 +108,20 @@ def check(self, external_data):
 
         self._update_version(external_data, latest_version, abs_url)
 
+    @staticmethod
+    def _substitute_placeholders(template, version):
+        version_list = LooseVersion(version).version
+        number_of_fields = len(version_list)
+        for i in range(number_of_fields):
```

Comment:
You can use `enumerate` here to get both the value and its index, e.g.
```suggestion
        for i, ver in enumerate(version_list):
```